src/neuroglancer/datasource/deepzoom/README.md

```diff
@@ -0,0 +1,11 @@
+The `"deepzoom"` data source format is based on static collections of files served directly over
+HTTP; it therefore can be used without any special serving infrastructure.  In particular, it can be
+used with data hosted by a cloud storage provider like Google Cloud Storage or Amazon S3.  Note that
+it is necessary, however, to either host the Neuroglancer client from the same server or enable CORS
+access to the data.
```

Comment:
Is there a linkj to a description of the deepzoom metadata format that could be added?

Reply:
Well, this is what Microsoft has from 2011: https://learn.microsoft.com/en-us/previous-versions/windows/silverlight/dotnet-windows-silverlight/cc645077(v=vs.95) - there's an XML Schema to download, and a page describing its elements, rather terrible for a documentation.

Personally I learned the format from https://www.openzoom.org/ ("Behind the scenes" at the bottom of the page), the [second part](https://www.gasi.ch/blog/inside-deep-zoom-2) in particular.

Reply:
Readme is extended now.